src/settings.ts

```diff
@@ -245,12 +245,30 @@ export function load(): ISettings {
     };
 }
 
-export async function change(settingName: string, newValue: any, global: boolean = false): Promise<void> {
-    const configuration: vscode.WorkspaceConfiguration =
-        vscode.workspace.getConfiguration(
-            utils.PowerShellLanguageId);
+export async function getConfigurationTarget(settingName: string): Promise<vscode.ConfigurationTarget> {
```

Comment:
This is to get where the setting is currently set, yes?

A small comment would be appreciated

Reply:
Yes, it gets the scope (they call it `ConfigurationTarget`) from where the _effective_ setting is coming from (workspace settings override user settings, hence why the order is important here). If someone defined the old setting on both levels, we only look at the one that determines the _effective_ setting and change that one.